modules/lvms-reusing-vg-from-prev-installation.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-reusing-vg-from-prev-installation_{context}"]
+= Reusing a volume group from the previous LVM Storage installation
+
+You can reuse an existing volume group (VG) from the previous {lvms} installation instead of creating a new VG.
+
+You can only reuse a VG but not the logical volume associated with the VG.
+
+[IMPORTANT]
+====
+You can perform this procedure only while creating an `LVMCluster` custom resource (CR).
+====
+
+.Prerequisites
+
+* The VG that you want to reuse must not be corrupted.
+* The VG that you want to reuse must have the `lvms` tag. For more information on adding tags to LVM objects, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/grouping-lvm-objects-with-tags_configuring-and-managing-logical-volumes#doc-wrapper[Grouping LVM objects with tags].
+
+.Procedure
+
+. Open the `LVMCluster` CR YAML file.
+
+. Configure the `LVMCluster` CR parameters as described in the following example:
++
+.Example `LVMCluster` CR YAML file
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: my-lvmcluster
+spec:
+# ...
+  storage:
+    deviceClasses:
+    - name: vg1  <1>
+      fstype: ext4 <2>
+      default: true
+      deviceSelector: <3>
+# ...
+        forceWipeDevicesAndDestroyAllData: true <4>
```

Comment:
Since, this LVMCluster CR example is mentioned in re-use VG section, we can either set line 45 as `forceWipeDevicesAndDestroyAllData: false <4>` OR remove the forceWipe option itself from this example. 

@suleymanakbas91  WDYT?

Reply:
@radeore 
Yes. It makes sense to have `forceWipeDevicesAndDestroyAllData: false `